src/main/java/frc/robot/Constants.java

```diff
@@ -86,12 +86,15 @@ public final class Constants {
       public static final double FLOOR_ARM_P_COEFF = 2;
       public static final double FLOOR_ARM_I_COEFF = 0;
       public static final double FLOOR_ARM_D_COEFF = 0;
-      public static final double FLOOR_ARM_OUTSIDE_ROBOT = 0.05451;
+      public static final double FLOOR_ARM_GROUND_INTAKE = 0.05451;
       public static final double FLOOR_ARM_INSIDE_ROBOT = 0.74;
-      public static final double FLOOR_ARM_MAX_POSITION = 0.68878;
+      public static final double FLOOR_ARM_MIDDLE = 0.68878;
+      public static final double FLOOR_ARM_HUMAN_PLAYER = 0;
+      public static final double FLOOR_ARM_MAX_POSITION = 0;
       public static final double FLOOR_ARM_MIN_POSITION = 0;
 
       //floor intake constants
       public static final double FLOOR_INTAKE_SPEED = 1;
+      public static final double FLOOR_INTAKE_SLOW_SPEED = 0.5;
```

Comment:
It might be easiest to have these as a NORMAL speed and HIGH speed, this would allow us to use the same button to score a cube for low and mid (the most common use case) and to have a separate button for high scoring.

It becomes a little complicated for the driver if we want to use high speed for low and high but slow speed for mid. Therefore easiest to think of it as a normal speed and high speed.